docs/deployments/testnet.md

```diff
@@ -57,7 +96,7 @@ title: Testnet
 | ERC20Mintable | [0xC138A7C89C357d8FA5A9b7CE775e612b766153e7](https://mumbai.polygonscan.com/address/0xC138A7C89C357d8FA5A9b7CE775e612b766153e7) |
 | TwabController | [0xf8ced175e56318FbAe092dBD112544B8D6D78777](https://mumbai.polygonscan.com/address/0xf8ced175e56318FbAe092dBD112544B8D6D78777) |
 | PrizePool | [0xA32C8f94191c9295634f0034eb2b0e2749e77974](https://mumbai.polygonscan.com/address/0xA32C8f94191c9295634f0034eb2b0e2749e77974) |
-| Claimer | [0x6a339df45f9c09fabb6df4f5cbcb17db2b2a3c7d](https://mumbai.polygonscan.com/address/0x6a339df45f9c09fabb6df4f5cbcb17db2b2a3c7d) |
+| Claimer | [0xC1aed8978B36af7Fb99E87ac7B399fECed47eeAD](https://mumbai.polygonscan.com/address/0xC1aed8978B36af7Fb99E87ac7B399fECed47eeAD) |
```

Comment:
Looks like the Claimer was re-deployed but the [mumbai-contracts.json](https://github.com/pooltogether/v4-docs/blob/main/data/mumbai-contracts.json) file was not updated. So you can revert this change.

Reply:
Ah, got it. Fixed with: [a93e0dc](https://github.com/pooltogether/v4-docs/pull/59/commits/a93e0dc7de7dd5d50c6f6e2785deb7989aa1e8e4)